src/tracer.js

```diff
@@ -125,7 +126,25 @@ export default class Tracer {
     *        to represent time values with sub-millisecond accuracy.
     * @return {Span} - a new Span object.
     **/
-    startSpan(fields: startSpanArgs): Span {
+    startSpan(operationName: string, fields: startSpanArgs): Span {
+        // Convert fields.childOf to fields.references as needed.
+        fields = fields || {};
+        fields.operationName = operationName;
+
+        if (fields.childOf) {
+            // Convert from a Span or a SpanContext into a Reference.
+            let childOf = opentracing.childOf(fields.childOf);
+            if (fields.references) {
+                fields.references.push(childOf);
+            } else {
+                fields.references = [childOf];
+            }
+            delete(fields.childOf);
```

Comment:
also, doesn't seem to make sense in light of L155. I suggest rewriting it into a single piece of logic, skipping the call to `opentracing.childOf(fields.childOf)` altogether